reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -55,9 +57,86 @@ export default function Quota() {
     );
   }
   return (
-    <Grid columns={2}>
-      {renderPieChart("CPU", quota.cpu)}
-      {renderPieChart("HDD", quota.disk)}
-    </Grid>
+    <>
+      <Grid columns={2}>
+        {renderPieChart("CPU", quota.cpu)}
+        {renderPieChart("Disk", quota.disk)}
+      </Grid>
+      <div className={styles["quota-info"]}>
+        {getQuotaNotifications(quota, config.adminEmail).map(
+          (notification, index) => {
+            const message = (
+              <span>
+                {notification.body} Please see:{" "}
+                <a
+                  href={REANA_QUOTAS_DOCS_URL}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {REANA_QUOTAS_DOCS_URL}
+                </a>
+              </span>
+            );
+            return (
+              <div key={index}>
+                <Notification
+                  icon={notification.icon}
+                  header={notification.header}
+                  message={message}
+                  error={notification.type === "critical"}
+                  warning={notification.type === "warning"}
+                  closable={false}
+                />
+              </div>
+            );
+          }
+        )}
+      </div>
+    </>
   );
 }
+
+function getQuotaNotificationInfo(quotaType, exceededLimit, adminEmail) {
+  if (quotaType === "disk" && exceededLimit)
+    return "Please delete unnecessary workflows or files.";
+  if (quotaType === "disk" && !exceededLimit)
+    return "Please consider liberating disk space by deleting some workflows or files.";
+  if (quotaType === "cpu" && exceededLimit)
+    return (
+      <span>
+        Please contact{" "}
+        {adminEmail ? (
+          <a href={`mailto:${adminEmail}`}>REANA administrators</a>
+        ) : (
+          "REANA administrators"
+        )}{" "}
+        for quota limit increase.
+      </span>
+    );
+}
+
+export function getQuotaNotifications(quota, adminEmail) {
+  let notifications = [];
+  for (const [key, value] of Object.entries(quota)) {
+    const { usage, limit, health } = value;
+    if (health !== "critical") continue;
+    const percentage = Math.round((usage?.raw / limit?.raw) * 100);
+    const exceededLimit = percentage >= 100;
+    const isDisk = key === "disk";
+    notifications.push({
+      header: `${isDisk ? "Disk" : "CPU"} quota ${
+        exceededLimit ? "exceeded" : "usage"
+      }`,
+      icon: isDisk ? "disk" : "microchip",
+      type: exceededLimit ? "critical" : "warning",
```

Comment:
What do you think about using `error` instead of `critical`? By doing so, we use the same terminology of the `Notification` component.

Reply:
I've tried to use the same terminology as defined [here](https://github.com/reanahub/reana-commons/blob/master/reana_commons/config.py#L430) and [here](https://github.com/reanahub/reana-ui/blob/031d6dc21163285e7374624c87bf18dfd5287712/reana-ui/src/util.js#L39) and to reuse the type to select the icon color. Indeed if we go for the grey color it could be changed!

Reply:
My only concern is that, as you said in https://github.com/reanahub/reana-ui/pull/275#discussion_r892324346, this is not coherent with what we get from the server. If the quota status returned by the server is `warning` no notifications are shown, while if `critical` is received it can become either a `warning` or `critical` notification depending on whether we have exceeded the limit or not.

I think we should strive to keep a one-to-one correspondence between what we receive from the server and the notifications. Otherwise, if we are not careful we might get into situations were reana-ui and reana-client report different health statuses to the user, which is confusing. This would also help in having code easier to maintain: if we decide to change the healthy/warning/critical thresholds, we only have to do it on the server-side and not also in reana-ui.

What do you think? How should we proceed?

Reply:
Totally agree that it's better to keep consistent to what we receive from the server side! One easy way to achieve it would be to simply render all critical quota notifications (>=85 percent) in red. It would be in line with what we do for quota pie chart labels. Example:
![Screenshot from 2022-06-08 16-35-12](https://user-images.githubusercontent.com/4990025/172645098-981dd154-1fde-48f9-8b49-48d304222271.png)
Right now we would show it like this:
![Screenshot from 2022-06-08 16-43-45](https://user-images.githubusercontent.com/4990025/172645976-173c7230-c2e9-4dfb-b1ac-5419e8cdd16d.png)
As you can see it's not matching pie chart label color.. It's kind of nice to distinguish between exceeded and non exceeded notification messages, but we should probably add this new level to server side to keep it consistent everywhere. cc @tiborsimko

Reply:
So I've changed the way we calculate health status in [reana-db](https://github.com/reanahub/reana-db/pull/171). Now it's fully consistent and looks like this:
![Screenshot from 2022-06-10 11-28-15](https://user-images.githubusercontent.com/4990025/173036195-bbd644da-5254-467a-aa7c-0b2ae4c1aaf9.png)
![Screenshot from 2022-06-10 11-28-53](https://user-images.githubusercontent.com/4990025/173036253-783c0da4-57c7-4961-bcc0-cd0d40092d79.png)